recipes/py-xxhash/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "py-xxhash" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/x/xxhash/xxhash-{{ version }}.tar.gz
+  sha256: 58ca818554c1476fa1456f6cd4b87002e2294f09baf0f81e5a2a4968e62c423c
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

To [link to xxhash you have to set `XXHASH_LINK_SO`](https://github.com/ifduyue/python-xxhash/blob/master/setup.py#L15-L19):
```suggestion
  script:
    - export XXHASH_LINK_SO=1  # [unix]
    - SET XXHASH_LINK_SO=1  # [win]
    - {{ PYTHON }} -m pip install . -vv
```